src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="
```

Comment:
should be in one line
<img width="609" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/f5f0a03e-fc9f-48cd-bcf1-b9e0c07141d0">